logrotate.c

```diff
@@ -1361,14 +1361,14 @@ int prerotateSingleLog(struct logInfo *log, int logNum, struct logState *state,
 	 * Construct the glob pattern corresponding to the date format */
 	dext_str[0] = '\0';
 	if (log->dateformat) {
-		i = j = 0;
+		size_t ii = 0, jj = 0;
```

Comment:
I like moving of the definitions of ```i``` and ```j``` to the innermost block but their rename is unnecessary.  It can be avoided by moving the definition of the remaining ```i``` variable to the innermost block, too.